docs/wiki/00 - SERVER INSTALL AND CONFIGURATION/01 - LORIS Install/Troubleshooting.md

```diff
@@ -0,0 +1,66 @@
+# Troubleshooting Common Installation Errors
+
+This guide assumes installation on any supported OS, and was primarily written based on an Ubuntu 18 stack.  
+
+This guide offers support for common issues faced when first installing LORIS. It is assumed that all installation steps provided have been 
+correctly followed and that all required dependencies have been installed.
+
+## Installation Roadblocks
```

Comment:
```suggestion
## MySQL 8 and PHP 7.3
```

This whole document is basically about installation roadblocks so this can be made more specific.